cli/src/templates/packages/expo-router/drawer/app/_layout.tsx.ejs

```diff
@@ -1,47 +1,48 @@
 import 'react-native-gesture-handler';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import { Slot } from 'expo-router';
-<% if (props.stylingPackage?.name === "tamagui") { %>
-	import React, { useEffect } from "react";
-	import { TamaguiProvider } from 'tamagui'
-	import { SplashScreen } from "expo-router";
-	import { useFonts } from "expo-font";
 
-	import config from '../tamagui.config'
+<% if (props.stylingPackage?.name==="tamagui" ) { %>
```

Comment:
all changes in this file seem to be code formatting?

Reply:
It really does seems like it, I was trying to setup my env and see how formatting was done.
Prettier didn't format it at all on save or manually, nor other formatters.
Can you share maybe extension you are using ?

This file was handled by copilot.
And it seems like it was following the configs correctly, especially the tab width ?
<img width="462" alt="Screenshot 2024-01-13 at 08 32 40" src="https://github.com/danstepanov/create-expo-stack/assets/30200380/ee6a757a-2c1e-45c5-8997-2f31f1efa712">

Reply:
interesting, I don't think these formatting instructions are being applied to .ejs files. I know it might be a bit annoying and we're working on better tools to work with ejs files but can you try to manually change the formatting stuff back? It will make it much easier to attribute your changes to actual functionality of this PR.